app/models/services/cbd_commitment_hash.rb

```diff
@@ -0,0 +1,34 @@
+class Services::CbdCommitmentHash
+  def initialize(cbd_hash, commitment)
+    @cbd_hash = cbd_hash
+    @commitment = commitment
+  end
+
+  def call
+    cbd_action = @cbd_hash["action"]
+    country = @cbd_hash["contacts"][0]["country"].present? ? Country.find_by(iso: @cbd_hash["contacts"][0]["country"]["identifier"].upcase)
+        : Country.find_by(iso: "--")
```

Comment:
This is doing quite a bit in a ternary. Would it be clearer in a standard `if`, possibly with some variables to make it clear what your accessing (e.g. why the first contact - is that meaningful?)

Reply:
i don't know, we've just inherited it. the API call has been sent to the DBD team, waiting on a response on whether that's the right place to get it and how to get the manager info

Reply:
have refactored it a bit and made a note